examples/grid2grid/main.py

```diff
@@ -0,0 +1,161 @@
+from evasdk import Eva
+from evaUtilities import *
+from progress.bar import *
+from config.config_manager import load_use_case_config
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+class EvaGrids:
+    def __init__(self, _eva, _plot_on_off):
+        names = config['grids']['names']
+        self.eva = _eva
+        self.plot = _plot_on_off
+        self.counter_max = {names[0]: config['grids']['row'][names[0]]*config['grids']['col'][names[0]]-1,

```

Comment:
Config should probably be passed in, having global state should be avoided

Reply:
modified, removed the global state